scripts/core/EXT_EventQueryKernelTimestamps.rst

```diff
@@ -64,8 +64,8 @@ This extension enables the querying of synchronized event timestamps.
     // Determine the level of support by getting the module properties
     ${x}DeviceGetProperties(hDevice, &devProps);
 
-    const bool supportsRaw = (0 != (tsProps.flags & ${X}_EVENT_QUERY_KERNEL_TIMESTAMPS_EXT_FLAG_RAW));
-    const bool supportsSynchronized = (0 != (tsProps.flags & ${X}_EVENT_QUERY_KERNEL_TIMESTAMPS_EXT_FLAG_HOST_SYNCHRONIZED));
+    const bool supportsKernel = (0 != (tsProps.flags & ${X}_EVENT_QUERY_KERNEL_TIMESTAMPS_EXT_FLAG_KERNEL));
+    const bool supportsSynchronized = (0 != (tsProps.flags & ${X}_EVENT_QUERY_KERNEL_TIMESTAMPS_EXT_FLAG_SYNCHRONIZED));
```

Comment:
minor : reword variables for better clarity? example "supportsKernel" -> "supportsKernelTs"